guide/onboarding/getting-to-know-your-users.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: guide
+title: Getting to know your users
+description: How best to understand and develop knowledge about your users.
+parent: Onboarding
+nav_order: 1
+permalink: /guide/onboarding/getting-to-know-your-users/
+main_classes: -no-top-padding
+---
+
+{% include picture.html
+   image = "/assets/images/guide/onboarding/getting-to-know-your-users.svg"
+   alt-text = "Bitcoin, key and lock"
+   width = 1600
+   height = 600
+   layout = "full-width"
+%}
+
+# Getting to know your users
+
+When building onboarding experiences, it is important to understand and develop knowledge about your users.
+Within the Bitcoin ecosystem, there is a wide range in the level of expertise. Identifying who you are building for is crucial, as these user groups value and understand varying characteristics of coin management tools. For example, “rookies” often favor convenient, easy-to-use wallets, while well-versed bitcoin owners tend to prioritize security[^1].
```

Comment:
Yeah, this is predominately in reference to a wallet application, so I can see how this can be a bit confusing actually. I think it's trying to get the point across the way you manage coins within wallet apps is different e.g Muun Vs Wasabi as an example